docs/plugins/suspendmanager.rst

```diff
@@ -49,6 +49,9 @@ Options
 ``-s``, ``--skipblocking``
     Also resume jobs that may block other jobs.
 
+``-a``, ``--all``
```

Comment:
I'd call this `-f`, `--force` instead of "all" since it's doing something that may be undesirable